backend/app-kubernetes-service/src/main/kotlin/services/UtilizationService.kt

```diff
@@ -0,0 +1,141 @@
+package dk.sdu.cloud.app.kubernetes.services
+
+import com.github.jasync.sql.db.util.length
+import dk.sdu.cloud.app.kubernetes.services.volcano.VolcanoJob
+import dk.sdu.cloud.app.kubernetes.services.volcano.volcanoJob
+import dk.sdu.cloud.app.orchestrator.api.CpuAndMemory
+import dk.sdu.cloud.app.orchestrator.api.JobUtilization
+import dk.sdu.cloud.service.k8.*
+
+/**
+ * A service responsible for fetching utilization information
+ */
+class UtilizationService(
+    private val k8: K8Dependencies
+) {
+    suspend fun allocatable(): CpuAndMemory {
```

Comment:
I think `retrieveCapacity` would make more sense.

Reply:
Agreed, I had some trouble with that name myself, but couldn't think of the right word